src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
-package core.basesyntax;
-
-public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class SalaryInfo {
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");

```

Comment:
[If you create a formatter, make it a constant field](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field)